java/integTest/build.gradle

```diff
@@ -27,17 +27,25 @@ dependencies {
 
 def standaloneRedisPorts = []
 def clusterRedisPorts = []
+def redisVersion = ""
 
 ext {
-    extractPortsFromClusterManagerOutput = { String output, ArrayList dest ->
+    extractPortsFromClusterManagerOutput = { String output ->
+        var res = []
         for (def line : output.split("\n")) {
             if (!line.startsWith("CLUSTER_NODES="))
                 continue
 
             def addresses = line.split("=")[1].split(",")
             for (def address : addresses)
-                dest << address.split(":")[1]
+                res << address.split(":")[1]
         }
+        return res
+    }
+    extractRedisVersion = { String output ->
```

Comment:
Please add to the PR description why it is needed (info implementation differs on redis 6 and 7)